iroh-net/src/hp/netcheck.rs

```diff
@@ -116,6 +79,49 @@ impl fmt::Display for Report {
     }
 }
 
+/// Latencies per DERP Region.
+#[derive(Debug, Default, PartialEq, Eq, Clone)]
+pub struct RegionLatencies(HashMap<u16, Duration>);
+
+impl RegionLatencies {
+    fn new() -> Self {
+        Default::default()
+    }
```

Comment:
I'm never really sure, but I usually prefer having it.  Logically creating a new struct is something different from just wanting a struct's default, at least to me, and it's nice to be able to do both.  The stdlib does this in lots of places as well, e.g. `String`.

Also clippy has https://rust-lang.github.io/rust-clippy/master/index.html#/new_without_default in which they suggest to have both rather than just switching entirely to default.

So my conclusion is generally that a new method without parameters is nice and certainly not un-idiomatic in Rust.

Reply:
having both seems fine to me